src/rdf4cpp/rdf/datatypes/registry/util/LangTag.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef RDF4CPP_REGISTRY_UTIL_LANGTAG_HPP
+#define RDF4CPP_REGISTRY_UTIL_LANGTAG_HPP
+
+#include <string_view>
+
+namespace rdf4cpp::rdf::datatypes::registry::util {
+
+struct LangTagCharTraits : std::char_traits<char> {
```

Comment:
typedef and using aliases being available in subclasses is non-standard behavior. You should redefine them, e.g. with 
```
using char_type = typename std::char_traits<char>::char_type;
```